New schema change tutorial and other minor changes

alexfrancoeur opened the pull request:
* REST API cleanup (sentence case formatting)
* Getting started simplification with more one liners for the CLI
* New schema change tutorial

Preview: https://xata.io/next/preview-path?path=/docs-preview/xataio:schema-change-tutorial/
Link to old sidebar preview (current one seems to be broken): https://xata-website-57r73n7os-xata.vercel.app/docs-preview/xataio%3Aschema-change-tutorial/getting-started

tsg commented on docs/tutorials/schema-change.mdx:
````diff
@@ -0,0 +1,165 @@
+Create and initialize a directory for your migrations (e.g., `.xata/migrations`) using the following command:
+
+```sh
+xata roll init --postgres-url `xata branch url`
````
I don't think we need the postgres url, the CLI already knows it.

```suggestion
xata roll init
```

tsg commented on docs/tutorials/schema-change.mdx:
````diff
@@ -0,0 +1,165 @@
+```sql
+ALTER TABLE products ADD COLUMN description TEXT;
+ALTER TABLE products DROP COLUMN name;
+```
````
This I'm not sure how it should work. Normally migrations in `.xata/migrations` are YAML files, not sql. But maybe the intention is to run `xata convert`?

tsg replied:
Something like this perhaps:

```
xata roll convert .xata/migrations/001_alter_products.sql > .xata/migrations/001_alter_products.json
```

I also raised a small issuse: https://xata-hq.slack.com/archives/C05TR0HPQ9Y/p1748521358166269

alexfrancoeur replied:
I originally had it as JSON / YAML and reverted. I'll just add commands for creating the actual migration files. I must have been tired when drafting this, I definitely was thinking that pgroll could diff and create the files a la Xata Lite.

alexfrancoeur commented on docs/tutorials/schema-change.mdx:
````diff
@@ -0,0 +1,165 @@
+WITH o AS (INSERT INTO orders DEFAULT VALUES RETURNING id) INSERT INTO order_items(order_id,product_id,qty) SELECT o.id,pid,(1+floor(random()*3))::int FROM o,(SELECT id pid FROM products ORDER BY random() LIMIT 5)p;
+```
+
+## 6. Create a schema change migration
````
I thought it was created during the roll init, but if not, yep -- makes sense to swap. I'll update.

alexfrancoeur commented:
@divyenduz @tsg I updated to remove `psql` DDL changes and added a few other improvements to the docs. If it looks good to you as an first take, mind giving a ✅?

alexfrancoeur commented:
Thanks @divyenduz, merging. We can iterate as needed.
